beginner_source/onnx/onnx_registry_tutorial.py

```diff
@@ -0,0 +1,451 @@
+# -*- coding: utf-8 -*-
+
+"""
+`Introduction to ONNX <intro_onnx.html>`_ ||
+`Export a PyTorch model to ONNX <export_simple_model_to_onnx_tutorial.html>`_ ||
+**Introduction to ONNX Registry**
+
+Introduction to ONNX Registry
+===========================
+
+**Authors:** Ti-Tai Wang (titaiwang@microsoft.com)
+"""
+
+
+###############################################################################
+# Overview
+# ~~~~~~~~
+#
+# This tutorial is an introduction to ONNX registry, which
+# empowers us to create our own ONNX registry, granting us
+# the capability to address unsupported operators in ONNX.
+#
+# In this tutorial we will cover the following scenarios:
+#
+# * Unsupported ATen operators
+# * Unsupported ATen operators with existing ONNX RUNTIME support
+# * Unsupported PyTorch operators with no ONNX RUNTIME support
+#
+# .. note::
+#
+#   This tutorial requires PyTorch 2.1.0 or later.
```

Comment:
Actually I think this is a great note for the `intro_onnx.py` tutorial, which goes through the onnx exporter dependencies (for the dynamo based version)